### PR TITLE
Update agent_builder.py

### PR DESCRIPTION
**Pull request by kinqsradio**
Update Agent Builder support local builders

### CODE REVIEW

**Review by kinqsradio — COMMENTED** [review]
Adding suffix, prefix.
This test on llama.cpp with mixtral 7b

File: autogen/agentchat/contrib/agent_builder.py [hunk 1]
```diff
@@ -762,3 +824,41 @@ def load(
                 }
             )
             return self._build_agents(use_oai_assistant, **kwargs)
+
+# Test
```
Comment (JieyuZ2) [inline autogen/agentchat/contrib/agent_builder.py:828]: could you move this test to `https://github.com/microsoft/autogen/blob/main/test/agentchat/contrib/test_agent_builder.py`?

**Comment by JieyuZ2** [comment]
> @kinqsradio please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

hey @kinqsradio could you reply to the @microsoft-github-policy-service by following the instruction here? It is necessary for contributing to the autogen

**Review by LinxinS97 — CHANGES_REQUESTED** [review]

**Comment by gagb** [comment]
Closing because of inactivity.

**Pull request closed without merging**